core/src/main/java/nl/nn/adapterframework/http/rest/ApiListener.java

```diff
@@ -72,6 +72,7 @@ public enum HttpMethod {
 
 	private @Getter String messageIdHeader = AppConstants.getInstance(getConfigurationClassLoader()).getString("apiListener.messageIdHeader", "Message-Id");
 	private @Getter String headerParams = null;
+	private @Getter String contentDispositionHeaderSessionKey;
```

Comment:
It might also be useful to be able to set the contentDispositionHeader from a fixed value, without having to set a session variable

Reply:
Lets stay in scope of the requested feature.

Reply:
Alsof het meer dan 5 minuten werk is

Reply:
Los van het feit dat niemand altijd een file met de zelfde naam wilt downloaden, en ik hier het nut niet van in zie, hadden we afgesproken dat dit een quick fix is.